usaspending_api/references/tests/test_legal_entity.py

```diff
@@ -0,0 +1,17 @@
+import pytest
+
+from usaspending_api.references.models import LegalEntity
+
+
+@pytest.mark.django_db
+def test_get_or_create_by_duns():
+    # Null duns should always return True for created
+    assert LegalEntity.get_or_create_by_duns(duns=None)[1]  # Assert created is True
+    assert LegalEntity.get_or_create_by_duns(duns=None)[1]  # Assert created is True
+    assert LegalEntity.get_or_create_by_duns(duns=None)[1]  # Assert created is True
```

Comment:
Great test--what do you think about adding a line for the `duns=""` case that you handle in `get_or_create_by_duns`?